django_irods/icommands.py

```diff
@@ -15,8 +15,8 @@ def __init__(self, exitcode, stdout, stderr):
                                                "Error processing IRODS request: {exitcode}. "
                                                "stderr follows:\n\n{stderr}".format(
                                                    exitcode=exitcode, stderr=stderr))
-        self.stdout = stdout
-        self.stderr = stderr
+        self.stdout = str(stdout)
+        self.stderr = str(stderr)
```

Comment:
at least one test failed because stdout is already a string.  I'm rerunning it without decode or str to see if there are situations where stdout or stderr aren't strings.

Reply:
Tests fail if I remove str() or use decode(), so I'm going to use str().  The alternative is to update usage so it's consistent... do you have a strong opinion?